src/Calendar.js

```diff
@@ -786,10 +785,6 @@ class Calendar extends React.Component {
      this.props.onView(view)
  };
 
- handleSelectAvailability = (...args) => {
```

Comment:
Was this removal intentional? Correct me if I'm wrong, but I don't recall this change being part of the initial PR. It also doesn't seem to follow from any subsequent PR comments.

Reply:
Was it just something that was part of the ticket's ask but missed in the initial PR?

Reply:
Yes intentional. This was part of a previous commit.

Reply:
Sorry, part of a previous ticket/PR - [RM-48531](https://elationhealth.atlassian.net/browse/RM-48531)